### DISCUSSION TITLE
newbie q: how to edit recurrence

### DISCUSSION THREAD

**Discussion by panmeri** [TechbeeAT/jtxBoard#1189] [Q&A]
Hi and thanks for this beautiful software.
I am new to jtx board, so I guess there is some learning curve to it.
I stumbled into this recurrence theme and saw some articles/q&a about it, but cannot find the answer to my question.
How can I edit the recurrence of a task?
E.g. I set up a task with monthly recurrence but forgot to set the end (like end after 10 recurrences).
I know that I can filter the display of future recurrences, but how do I set this "end after x recurrences" or "end by reaching date" or something like this?

There is a button "go to series" but I cannot figure out how to limit the series.
There is the button "detach from series" giving me an caveat and as I don't want to mess things up ...

Thanks for a word of clarification.
Kind regards, Igor

**Comment by patrickunterwegs** [comment]
Hey @panmeri !

When you go to the series click on the pencil to edit the whole series again. Then scroll down to the recurrence section. There you can clock in "ends never" and change the option 😉

![Screenshot_20240126_151828_jtx Board.jpg](https://github.com/TechbeeAT/jtxBoard/assets/72232737/405970f3-ee8f-4c75-bbd9-7f22015a05ea)
